### PR TITLE
Molecule 2

### PR DESCRIPTION
**Pull request by manics**
I'm not sure if we still need this role but it's easy enough to update to molecule 2.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -2,47 +2,43 @@
...
+  become: true
   template:
     dest: /etc/systemd/system/cadvisor.service
     src: systemd-system-cadvisor-service.j2
   notify:
-  - reload systemd
-  - restart cadvisor
-
-# Flush handlers to ensure systemd is reloaded
-- name: cadvisor | flush systemd handlers
```
Comment (sbesson) [inline tasks/main.yml:40]: Is that now unnecessary due to the handler changes?

**Review by sbesson — COMMENTED** [review]
One inline question but otherwise looks good. Does the usage of the `systemd` module rather than `system` make it worth tagging this as `0.4.0` or `0.3.2`?

File: tasks/main.yml (hunk 1 above)
Comment (manics) [reply tasks/main.yml:40]: This was a workaround to reload the systemd config, but this is now built in to the `systemd` module (`daemon_reload: true`)

**Comment by manics** [comment]
This role only supports Centos 7, so the `service` module uses the `systemd` module internally. `0.3.2` sounds fine to me.

**Review by sbesson — APPROVED** [review]

**Pull request merged**